gadget/install/install.go

```diff
@@ -377,6 +376,140 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, bootDevice string, options
 	}, nil
 }
 
+// structureFromPartDevice returns the OnDiskStructure for a partition
+// node.
+func structureFromPartDevice(diskVol *gadget.OnDiskVolume, partNode string) (*gadget.OnDiskStructure, error) {
+	for _, p := range diskVol.Structure {
+		// We remove /dev from p.Node here
```

Comment:
The comment can now be removed, it only applied to the old code which was doing `filepath.Base(p.Node) == partDevice` here

Reply:
Good catch, thanks!